### ISSUE TITLE
Error Messages for AdNauseam Android

### ISSUE THREAD

**Issue by cqx931** [dhowe/AdNauseam#1715]
<img width="1151" alt="image" src="https://user-images.githubusercontent.com/3056154/90663272-6da6e200-e217-11ea-8bda-e08b6222a367.png">

**Comment by cqx931** [comment]
browser.windows not supported for firefox android https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows

**Comment by cqx931** [comment]
This means, we can no longer use`browser.windows.onRemoved`([see](https://github.com/dhowe/AdNauseam/blob/master/src/js/adn/core.js#L2034)) to delete private ads on mobile for private mode. 

As an alternative, we can remove private ads at `validateAdStorage()`([see](https://github.com/dhowe/AdNauseam/blob/master/src/js/adn/core.js#L90)) during initialization.

**Comment by dhowe** [comment]
So we have to mark them as hidden, then delete them the next time the extension starts up?

**Comment by cqx931** [comment]
@dhowe yes exactly. Private ads removal on validateAdStorage() should also applies to desktop when the browser is shutdown.

**Comment by cqx931** [comment]
after fixing browser.windows.onRemoved(), the error messages are fixed for android
